frontend/src/scenes/scenes.ts

```diff
@@ -428,6 +428,7 @@ export const redirects: Record<
         return urls.replay()
     },
     '/replay': urls.replay(),
+    '/replay/recent': urls.replay(),
```

Comment:
I'm assuming this does the redirect and maintains any query params?

Reply:
obviously I'd tested that 🙈